ui/redux/actions/claims.js

```diff
@@ -186,6 +186,21 @@ export function doResolveUri(
   return doResolveUris([uri], returnCachedClaims, resolveReposts, additionalOptions);
 }
 
+export function doGetClaimFromUriResolve(uri: string) {
+  return async (dispatch: Dispatch) => {
+    let claim;
+    await dispatch(doResolveUri(uri, true))
+      .then((response) =>
+        Object.values(response).forEach((resposne) => {
+          claim = resposne;
+        })
+      )
+      .catch((e) => {});
+
+    return claim;
+  };
+}
```

Comment:
Few minor concerns from a (re)usability standpoint:
- [ ] The function assumes the caller has checked that we have not resolved before, because `response` would be `undefined` if previously resolved.  Can either fix by setting the parameter to `false`, or just document this gotcha behavior.
- [ ] `resposne` typo
- [ ] But why `forEach`?  Assuming that it is possible for `resolve` to return multiple entries for 1 `uri`, setting `claim` directly to the last entry would be a faster equivalent than going through everything.

Reply:
- [ ] I believe the first checkbox above is causing https://sentry.lbry.tech/organizations/odysee/issues/28402, where `doGetClaimFromUriResolve` is still returning `undefined` despite the effort to resolve.
 
- [ ] On a related note, https://sentry.lbry.tech/organizations/odysee/issues/28455 is also failing from a `null` claim and would probably require the same solution.  However, since it is `null`, does that mean the claim is already abandoned?